embassy-rp/src/pio/mod.rs

```diff
@@ -749,7 +749,7 @@ impl<'d, PIO: Instance + 'd, const SM: usize> StateMachine<'d, PIO, SM> {
                     w.set_set_count(1);
                 });
                 // SET PINDIRS, (dir)
-                unsafe { sm.exec_instr(0b111_00000_100_00000 | dir as u16) };
+                unsafe { sm.exec_instr(0b1110_0000_1000_0000 | dir as u16) };
```

Comment:
I guess the format was chosen intentionally, to match the PIO instructions (3-5-3-5 bits).
Chapter 3.4.1 https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf